pontoon/base/tests/managers/test_entity.py

```diff
@@ -930,3 +930,49 @@ def test_mgr_entity_reset_active_translations(resource_a, locale_a):
     active = entities[4].translation_set.filter(active=True)
     assert active[0].string == entities[4].string + " translation1"
     assert active[1].string == entities[4].string_plural + " translation1plural"
+
+
+@pytest.mark.django_db
+def test_get_word_count(resource_a, locale_a):
+    """ How many words in a string
+    """
+    testEntitiesQuerySet = Entity.for_project_locale(resource_a.project, locale_a)
+    count = testEntitiesQuerySet._get_word_count("String 123 =+& string hh-gg object.string")
+    assert count == 5
+
```

Comment:
Hello @mathjazz ! Could you please review this PR again?

Do we need one more test for get_word_count for empty string?